docs/nodes/validator-node/operator/node-requirements.md

```diff
@@ -2,111 +2,151 @@
+independent machines. These machines should be well-provisioned, meet the requirements outlined below and
+be isolated from each other. Maintaining resource isolation between the validator and the VFN is important
+for security and to ensure that the nodes do not encounter performance degradation, instability or failures when
+under load.
+
+:::tip Terraform support
+For deploying the validator and VFN in the cloud, we provide Terraform support on two cloud providers: **GCP** and **AWS**. See [**Running Validator Node**](running-validator-node/index.md).
+:::
 
 ## Hardware requirements
 
-For running an Aptos **validator node and validator fullnode** we recommend the following hardware resources:
-For running a validator and validator fullnode is that your hardware spec should be good enough to achieve a 30k peak
-TPS on mainnet.
+For running an Aptos validator and VFN in mainnet, we recommend that your hardware be performant enough to maintain
+~30,000 transactions per second (TPS). There are two ways to evaluate if your hardware meets this requirement:
 
-In order to evaluate that, checkout aptos-core repo and install required dependencies (See [**cloning aptos-core**](https://aptos.dev/guides/building-from-source#clone-the-aptos-core-repo)).
-Use following commands to run the performance benchmark
+1. Use the reference specs provided below.
+1. Run the performance benchmarking tool provided by Aptos.
 
-```
-TABULATE_INSTALL=lib-only pip install tabulate
-./testsuite/performance_benchmark.sh --short
-```
+Note that both the validator and VFN require sufficient hardware separately (i.e., two separate machines that
+satisfy the requirements outlined below).
 
-Once the script finishes, it will print out a table, and column "t/s" is your achieved "transactions per second". Evaluation criteria is encoded in the script, and will warn you after the table if you've not met any of the criteria.
+### Reference specs
 
-If you don't want to run the benchmark tool for hardware evaluation, you can use the following reference specs
+The reference specs for running an Aptos validator and VFN on mainnet can be seen below:
 
-- **CPU Requirement Reference**:
-  - 32 cores
-  - 2.8GHz, or faster
-  - AMD Milan EPYC or Intel(R) Xeon(R) Platinum
+- **CPU**: 32 cores, 2.8GHz, or faster, AMD Milan EPYC or Intel(R) Xeon(R) Platinum
 - **Memory**: 64GB RAM.
 - **Storage**: 2T SSD with at least 60K IOPS and 200MiB/s bandwidth.
 - **Networking bandwidth**: 1Gbps
-- **Example machine types on various clouds**:
-  - **AWS**
-    - c6id.16xlarge (if using a local SSD)
-    - c6i.16xlarge + io1/io2 EBS volume with 60K IOPS.
-  - **GCP**
-    - t2d-standard-60 + pd-ssd with 60K IOPS.
 
-### Motivations for hardware requirements
+Example machine types that meet the reference specs (on various clouds):
+
+- **AWS**
+  - c6id.16xlarge (if using a local SSD).
+  - c6i.16xlarge + io1/io2 EBS volume with 60K IOPS.
```

Comment:
Let's just mention `io2` since `io1` is effectively worse in any way and deprecated.